src/validators.js

```diff
@@ -41,6 +41,14 @@ export const documentContainer = value => value instanceof DocumentContainer
 export const noMoreThanAvailableOnBalance = balance => value => {
   return +balance > +value
 }
+export const maxDecimalDigitsCount = maxDecimalDigitsCount => value => {
```

Comment:
mb move to amount validator? or at least use it inside it

Reply:
maybe in the future